sklearn/preprocessing/tests/test_data.py

```diff
@@ -2475,14 +2476,19 @@ def test_standard_scaler_sparse_partial_fit_finite_variance(X_2):
     assert np.isfinite(scaler.var_[0])
 
 
-@pytest.mark.parametrize("feature_range", [(0, 1), (-10, 10)])
-def test_minmax_scaler_clip(feature_range):
+@pytest.mark.parametrize(
+    "array_namespace, device, _", yield_namespace_device_dtype_combinations()
+)
+@pytest.mark.parametrize("feature_range", [(0.0, 1.1), (-10.0, 10.0)])
+def test_minmax_scaler_clip(feature_range, array_namespace, device, _):
```

Comment:
Let's put `array_api` in the test name so that it can be picked up by the CUDA CI that [only runs tests with "array_api" in their name](https://github.com/scikit-learn/scikit-learn/blob/5600faab452b0adb1f4090219012aa3ae6208c66/.github/workflows/cuda-ci.yml#L45) to avoid spending GPU time running non-array API tests.

```suggestion
def test_minmax_scaler_clip_array_api(feature_range, array_namespace, device, _):
```